src/factories.ts

```diff
@@ -1,15 +1,13 @@
 import ProxyFactory111 from './assets/v1.1.1/proxy_factory.json'
 import ProxyFactory130 from './assets/v1.3.0/proxy_factory.json'
 import { DeploymentFilter, SingletonDeployment } from './types'
-import { applyFilterDefaults, findDeployment } from './utils'
-
-
+import { findDeployment } from './utils'
 
 // This is a sorted array (newest to oldest)
 const factoryDeployments: SingletonDeployment[] = [
     ProxyFactory130, ProxyFactory111
 ]
 
 export const getProxyFactoryDeployment = (filter?: DeploymentFilter): SingletonDeployment | undefined => {
-    return findDeployment(applyFilterDefaults(filter), factoryDeployments)
+    return findDeployment(filter, factoryDeployments)
```

Comment:
A default filter parameter in `findDeployment` is now used instead of `applyFilterDefaults`. It seemed like a worthwhile refactor, but I am happy to revert it.